cg/meta/demultiplex/status_db_storage_functions.py

```diff
@@ -55,6 +55,23 @@ def store_flow_cell_data_in_status_db(
     store.session.commit()
 
 
+def check_if_samples_have_files(flow_cell: FlowCellDirectoryData) -> None:
```

Comment:
How to name this function considering that it has no return and raises an error if the checks failed. I was thinking renaming to `validate_samples_have_files`

Reply:
I think the existing validation functions which behave in a similar way are named like `validate_demultiplexing_complete`. But I think this is a good name too :)

Reply:
Renamed to `validate_samples_have_fastq_files`